lib/board.rb

```diff
@@ -0,0 +1,54 @@
+#data and logic of the Tic Tac Toe game board
+#board state held in @cells(Array)
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "] #holds the current state of the game @cells
+  end
+
+  def reset!
+   self.cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "] #clears all tokens from the game @cells
+  end
+
+  def display
+    puts " #{@cells[0]} " + "|" + " #{@cells[1]} " + "|" + " #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} " + "|" + " #{@cells[4]} " + "|" + " #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} " + "|" + " #{@cells[7]} " + "|" + " #{@cells[8]} "
+  end
+
+  def position(pos_from_user)
+    position = pos_from_user.to_i - 1 #decreases num from user to match array indices
```

Comment:
as this variable is only used once you might as well just set up your method to look like this
```@cells[pos_from_user.to_i - 1]```
A variable would only be helpful if used more than once.